docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='using-standalone-perspectives-library-proc']
+= Using standalone library perspective
+
+You can use the library perspective of {CENTRAL} to select a project you want to edit. You can also perform all the authoring functions on the selected project. The home page of the standalone library perspective is the *Projects* tab of the *MySpace* page.
+
+Further the standalone library perspective can be used in two ways, _with_ and _without_ using the `header=UberfireBreadcrumbsContainer` parameter. The difference between the two ways is that the link with the *_header_* parameter will display a breadcrumbs link on top of the library perspective. Using this link you can create additional *Spaces* for your projects.
+
+.Procedure
+. Login to {CENTRAL}.
+. In a web browser, enter the appropriate link:
```

Comment:
instead of link, consider web address. Also, I looked up "URL" in the style guide and it says to avoid it. 

Basically, it says "either use "web address" or do not use a term at all. For example, write "go to".. `http://localhost:8080/business-central/kie-wb.jsp?standalone=true&perspective=LibraryPerspective&header=UberfireBreadcrumbsContainer

Reply:
Changed partially. 
Michele, I have changed "web link" to "web address" but have not rephrased the sentences using "go to ..." for readability consistency.